engines/mohawk/configure.engine

```diff
@@ -1,6 +1,6 @@
 # This file is included from the main "configure" script
-# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine mohawk "Mohawk" yes "cstime myst mystme riven" "Living Books" "highres"
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
+add_engine mohawk "Mohawk" yes "cstime myst mystme riven" "Living Books" "highres" "mt32emu"
```

Comment:
IIRC the MIDI code in Mohawk was removed some time ago since none of the games we support made use of it.

Reply:
Fixed